Modules/Core/Common/test/itkMetaDataDictionaryGTest.cxx

```diff
@@ -33,8 +33,8 @@ createMetaDataDictionary()
 {
   itk::MetaDataDictionary metaDataDictionary;
 
-  itk::EncapsulateMetaData<float>(metaDataDictionary, "one", static_cast<float>(1));
-  itk::EncapsulateMetaData<float>(metaDataDictionary, "two", static_cast<float>(2));
+  itk::EncapsulateMetaData<float>(metaDataDictionary, "one", 1.0f);
+  itk::EncapsulateMetaData<float>(metaDataDictionary, "two", 2.0f);
```

Comment:
Sorry, this seems beyond the scope of this commit.

Reply:
But you just  requested this sort of translation for `int`, so why not `float`?  Is there a case where this textual substitution could fail?

Reply:
This is somewhere in the middle of what looks like an automatic "bulk" of ~1000 changes that mostly add ` static_cast`'s. In order to approve the PR, I would have to check all those lines.

Reply:
I will revert these with the next commit.